data/json/professions.json

```diff
@@ -343,6 +343,7 @@
     "name": "Vagabond",
     "description": "Circumstance left you wandering the world, alone.  Now there is nothing to go back to, even if you wanted to.  Perhaps your experience in fending for yourself will prove useful in this new world.",
     "points": 2,
+    "proficiencies": [ "prof_cooking" ],
```

Comment:
```suggestion
    "proficiencies": [ "prof_cooking_basic" ],
```

Reply:
Their is not a prof_cooking_adv or prof_cooking this these make little logic to name

Reply:
Even if you aren't making a `prof_cooking_adv`, which I had hoped you would see that as a possibility based on this suggestion, if someone ever comes along and wants to add an advanced cooking proficiency they will be bound by the naming convention you set here.  So I felt like it was better to go ahead and name things in a way that expects there will be additions.

Reply:
Dont i also have to change all the "prof cooking" in the crafting to have basic a well?